ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml

```diff
@@ -3271,11 +3283,11 @@ tests:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-vsphere-upi-zones
 - as: vsphere-ipi-ovn-winc-f14
-  cron: 6 22 14 7 *
+  cron: 6 22 9,23 * *
```

Comment:
pls update the cron setting following https://github.com/rrasouli/release/blob/a70458f5302c54a2c5386d9ad8736bb17a28d3e0/ci-operator/config/openshift/openshift-tests-private/README.md#job-frequency